recipes/crossdb/all/conanfile.py

```diff
@@ -0,0 +1,50 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get
+from conan.tools.microsoft import is_msvc
+import os
+
+required_conan_version = ">=1.53.0"
+
+class CrossDBConan(ConanFile):
+    name = "crossdb"
+    description = "Ultra High-performance Lightweight Embedded and Server OLTP RDBMS✨"
+    license = "MPL-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/crossdb-org/crossdb"
+    topics = ("database", "oltp", "embedded")
+    package_type = "shared-library"
+    settings = "os", "arch", "compiler", "build_type"
+
+    def configure(self):
+        self.settings.rm_safe("compiler.cppstd")
+        self.settings.rm_safe("compiler.libcxx")
+
+    def layout(self):
+        cmake_layout(self, src_folder="src")
+
+    def validate(self):
+        if is_msvc(self):
+            raise ConanInvalidConfiguration(f"${self.ref} does not support MSVC")
+
+    def source(self):
+        get(self, **self.conan_data["sources"][self.version], strip_root=True)
+
+    def generate(self):
+        tc = CMakeToolchain(self)
+        tc.cache_variables["CMAKE_MACOSX_RPATH"] = True
+        tc.generate()
+
+    def build(self):
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def package(self):
+        copy(self, "LICENSE", self.source_folder, os.path.join(self.package_folder, "licenses"))
+        cmake = CMake(self)
+        cmake.install()
```

Comment:
```suggestion
        cmake.install()
        fix_apple_shared_install_name(self)
```

Instead of using CMAKE_MACOSX_RPATH